src/dv-convert-labels-to-mesh.cxx

```diff
@@ -0,0 +1,199 @@
+// Boost
+#include <boost/program_options.hpp>
+
+// ITK
+#include <itkMesh.h>
+#include <itkImageFileReader.h>
+#include <itkMeshFileWriter.h>
+#include <itkSTLMeshIO.h>
+#include <itkBinaryMask3DMeshSource.h>
+#include <itkImage.h>
+#include <set>
+#include <itkUnaryFunctorImageFilter.h>
+
+const unsigned int Dimension = 3;
+typedef float      TCoordinate;
+
+typedef unsigned char  PixelType;
+typedef itk::Image< PixelType, Dimension >   ImageType;
+
+typedef itk::Mesh<float,Dimension>                         MeshType;
+typedef itk::BinaryMask3DMeshSource< ImageType, MeshType >   MeshSourceType;
+
+typedef itk::Mesh< TCoordinate, Dimension > TMesh;
+typedef itk::MeshFileWriter< TMesh >        TWriter;
+
+namespace itk
+{
+
+  namespace Functor
+  {
+
+    template< class TInput, class TOutput>    class MyFilter 
+    {
+    public:
+      MyFilter() {};    ~MyFilter() {};
+      typedef typename NumericTraits<TInput>::RealType InputRealType;
+
+      inline TOutput operator()( const TInput & A )
+      {
+        // Checks the functLabels for the value. Returns 1 if included. 0 otherwise
+        return static_cast<TOutput>( functLabels.count(A) );
+      }
+
+      void SetLabels( std::set<float> value )
+      { functLabels = value; }
+
+    private:
+      std::set<float>  functLabels;
+
+    };
+  };
+
+  template<class TInputImage, class TOutputImage>
+  class MyFunctorImageFilter :
+  public UnaryFunctorImageFilter< TInputImage,TOutputImage,
+  Functor::MyFilter< typename TInputImage::PixelType,typename TOutputImage::PixelType> >
+  {
+  public:
+
+
+    typedef MyFunctorImageFilter          Self;
+    typedef UnaryFunctorImageFilter< ImageType, ImageType,
+    Functor::MyFilter< typename ImageType::PixelType,typename ImageType::PixelType > >  Superclass;
+
+    typedef SmartPointer< Self >          Pointer;  typedef SmartPointer< const Self >    ConstPointer;
+
+    itkNewMacro( Self );
```

Comment:
Do you see these random quarter notes everywhere at the ends of all the lines?  Haha, must be some weird encoding thing.